packages/forms/resources/css/components/select-action.css

```diff
@@ -0,0 +1,3 @@
+.fi-fo-field-wrp .fi-ac-select-action .fi-input-wrapper {
+    @apply bg-transparent shadow-none ring-0;
+}
```

Comment:
What's this needed for?

Reply:
> What's this needed for?

The SelectAction normally looks like a button. But it doesn't look good when as suffix / prefix action as a button. Neither did it look good in the hint. This just removes the background and outline.

All other prefix / suffix actions also don't have these styles applied by default.

Reply:
Hmm, I think it's better to remove the wrapper instead, as we're nesting wrappers now...

Reply:
Oh, yeah. That works much better!